roles/kubernetes-apps/metallb/tasks/main.yml

```diff
@@ -67,16 +67,6 @@
   register: result
   until: result is not failed
 
-- name: Kubernetes Apps | Install and configure MetalLB
-  kube:
-    name: "MetalLB"
-    kubectl: "{{ bin_dir }}/kubectl"
-    filename: "{{ kube_config_dir }}/{{ item.item }}"
-    state: "{{ item.changed | ternary('latest','present') }}"
-  become: true
-  with_items: "{{ rendering.results }}"
```

Comment:
metallb.yaml has the namespace creation and the CRDs. If this yaml does not get applied first then the subsequent manifests will fail

Reply:
Hmmm this variable shouldn't change the original order is it?